lib/log.ts

```diff
@@ -24,6 +24,18 @@ export class Log {
     this.info("Error:", err.toString());
   }
 
+  assertTrue(result: boolean, ...msgs) {
+    if (!result) {
```

Comment:
```
if (result !== true)
```

Reply:
I utterly dislike the style of this comparison, but somehow I am now expecting to see a JS trickery popping out of the rabbit hole... ^^'

Reply:
@tharvik took a long time bashing my nose on this, so I pass on the teaching ;) The method name is `assertTrue`, so it should make sure that the argument is `true`. So it should fail if it's `!== true`... It removes some of the 'niceness' of javascript, but I feel much more cozy with a nicely typed language.